README.md

```diff
@@ -6,12 +6,12 @@
 
 ## Description
 
-Derive macro to simplify deriving standard and other traits with custom
-generic type bounds.
+Attribute proc-macro to simplify deriving standard and other traits with
+custom generic type bounds.
```

Comment:
Since we moved to using an attribute macro instead of `derive`, we didn't update the wording. @ModProg I think it would be good if we also updated the description of the GitHub repo with the same text.

Reply:
good catch